appmap/pytest.py

```diff
@@ -35,9 +34,10 @@ def is_in_class(self):
 
     @property
     def feature_group(self):
-        return inflection.humanize(
-            inflection.titleize(self.class_name.removeprefix('Test'))
-        )
+        test_name = self.class_name
```

Comment:
@apotterri since we already have written code for Py3.9, I think rather than back porting the whole code, we should add backward compatibility with wrapper functions. Somewhat similar to six https://pypi.org/project/six/.

Each compatibility function could have notes about the minimum version. As and when we keep moving ahead with version of Python, it would be very easy to keep upgrading code. And it would reduce the technical debt.

Reply:
https://docs.python.org/3/whatsnew/3.9.html

> Python 3.9 is the last version providing those Python 2 backward compatibility layers, to give more time to Python projects maintainers to organize the removal of the Python 2 support and add support for Python 3.9.

> Aliases to Abstract Base Classes in the collections module, like collections.Mapping alias to collections.abc.Mapping, are kept for one last release for backward compatibility. They will be removed from Python 3.10.

Reply:
That seems like a reasonable approach. We're under some time pressure to support older versions of Python, though. At this point, I've made all the changes to move use back to 3.5, so I'm inclined to stick with these changes.